storey/utils.py

```diff
@@ -164,6 +164,7 @@ def update_in(obj, key, value):
 
 
 def hash_list(list_to_hash):
+    list_to_hash = [str(element) for element in list_to_hash]
```

Comment:
Looks like this is actually the [fastest way](https://waymoot.org/home/python_string/) to do it. ✅